corehq/apps/domain/models.py

```diff
@@ -369,7 +369,7 @@ def active_for_user(user, is_active=True):
         else:
             couch_user = CouchUser.from_django_user(user)
         if couch_user:
-            domain_names = couch_user.get_domains()
+            domain_names = couch_user.get_domains(for_ui=True)
```

Comment:
Lot of places call `active_for_user`, will change this latter to pass `for_ui` up the stack once we decide what parts of UI should Hide domain-list and what parts should show all domains (even hidden from UI)